rest/rest-api/src/main/java/org/apache/brooklyn/rest/api/AdjunctApi.java

```diff
@@ -234,4 +236,21 @@ public Response setConfig(
             @PathParam("config") String configKeyName,
             @ApiParam(name = "value", value = "New value for the configuration", required = true)
             Object value);
+    
+
+    @GET
+    @Path("/{adjunct}/activities")
+    @ApiOperation(value = "Fetch list of tasks for this adjunct")
+    @ApiResponses(value = {
+            @ApiResponse(code = 404, message = "Could not find application, entity, or adjunct")
+    })
+    public List<TaskSummary> listTasks(
+            @ApiParam(value = "Application ID or name", required = true) @PathParam("application") String applicationId,
+            @ApiParam(value = "Entity ID or name", required = true) @PathParam("entity") String entityId,
+            @ApiParam(value = "Adjunct ID or name", required = true) @PathParam("adjunct") String adjunctToken,
+            @ApiParam(value = "Max number of tasks, or -1 for all (default 200)", required = false) 
```

Comment:
I'm not a fan of `-1` for all. But it's already used in `EntityApi.listTasks` so I'll live with it.

Reply:
yes for me it is simply "least bad"